README.md

```diff
@@ -2,6 +2,38 @@
 
 Shared Prettier configuration for usage within the [fullstax Gmbh & CO. KG](https://www.fullstax.de).
 
+## Opinions
+
+We stick to the very opinionated [default options](https://prettier.io/docs/en/options.html) of the [current version of prettier](https://www.npmjs.com/package/prettier) but we have a few (strong) opinions of our own:
+
+### singleQuote
+
+We set `singleQuote: true`.
+
+We don't use jsx or other abominations, and also don't thing you should put long strings inside you code. So for us singel quotes are just more lightweight and improve readability when used together with string-templates, that we use for most strings.
+
+### useTabs
+
+We set `useTabs: true` and we have a very strong opinion about this!
+
+Why the hell would you use spaces for indentation? That's just very unprofessional! https://www.youtube.com/watch?v=SsoOG6ZeyUI
+
+### printWidth
+
+We use a `printWidth` of `100` instead of the default `80`
+
+While this is explicitly discurraged in [the prettier docs](https://prettier.io/docs/en/options.html#print-width) we acknowledge that screens are much wider than high, nowadays.
+
+We just prefer a little less broken up lines in our code. This is not a strong opinion but instead of putting a big warning-block inside their docs prettier should acknowledge that a lot of projects currently overwrite the default of 80! Not every developer is using vim...
```

Comment:
~~a little~~

I think the comment in the prettier docs is actually quite useful